vue-components/src/components/Popover.vue

```diff
@@ -179,5 +194,239 @@ $base: '.wikit-Popover';
 			transform-origin: bottom left;
 		}
 	}
+
+	&--bottom-right {
+		#{$base}__content-wrapper {
+			margin-block-start: $wikit-Popover-distance + $wikit-Popover-pointer-height;
+			inset-inline-start: 50%;
+			transform: translateX(-($wikit-Popover-pointer-width + $wikit-Popover-pointer-width / 2));
+			inset-block-start: 100%;
+		}
+
+		#{$base}__pointer {
+			width: $wikit-Popover-pointer-width;
+			height: $wikit-Popover-pointer-height;
+			inset-block-start: -$wikit-Popover-pointer-height;
+			inset-inline-start: $wikit-Popover-pointer-width;
+		}
+
+		#{$base}__pointer::before {
+			inset-block-end: 0;
+			transform-origin: bottom left;
```

Comment:
I think this `transform-origin: bottom left;` is identically for every position, right? Then it can move out of the positioning rules, into the `&__pointer::before` rule above, next to the transform rule that is already there.

Reply:
thanks. it was a silly oversight